framework/transport/ieee1905_transport/include/mapf/transport/ieee1905_transport.h

```diff
@@ -154,6 +154,21 @@ class Ieee1905Transport {
     };
     friend std::ostream &operator<<(std::ostream &os, const Packet &m);
 
+    // reliable multicast neighbours map
+    struct ieee1905_neighbor {
+        sMacAddr al_mac;
+        unsigned int if_index;
+        std::chrono::steady_clock::time_point last_seen;
+    };
+
+    std::unordered_map<sMacAddr, ieee1905_neighbor> neighbors_map_;
+
+    // According to the ieee1905.1 specification section 8.2.1.1, a 1905.1 management entity shall
+    // transmit a topology discovery message *at least* once every 60 seconds. Since any message from
+    // the neighbor will update it's last_seen, in practice it should be safe to consider it dead and
+    // delete it if 70 seconds have passed.
```

Comment:
Except that this is not actually a *neighbour* map, it's a map of known devices, neighbour or multi-hop. In fact, it's mainly the controller that should be in here, and if it's behind another repeater, it's quite likely that we don't hear from it every minute. A prplMesh controller will indeed send a topology query every minute (IIRC), but you can't rely on that.

In practice it's very unlikely that 1905.1 devices appear or disappear, so the timeout can be quite large.

Reply:
So do you think I should use a different name? how about ieee1905_known_devices?
Which timeout do you suggest?